tox.ini

```diff
@@ -6,11 +6,9 @@ deps =
     coverage
     pretend
     pytest
-    https://github.com/pypa/pip/archive/master.zip#egg=pip
-# The --ignore-installed install options is needed to install pip url
-# (needed to issue #95). This can be removed once pip 10.0 is out.
+    pip>=9.0.2
 install_command =
-    pip install {opts} {packages} --ignore-installed
+    pip install {opts} {packages}
```

Comment:
One last change: there's no need to override `install_command` anymore.

Reply:
I don't understand what you mean, please rephrase

Reply:
We can just use the default value for `install_command`, there's no need to override it anymore now that `--ignore-installed` is not necessary. Just remove those 2 lines:
```cfg
install_command =
    pip install {opts} {packages}
```

Reply:
great, got it